docs/hdfs_connector.rst

```diff
@@ -54,16 +82,37 @@ The three records entered are published to the Kafka topic ``test_hdfs`` in Avro
 
 Before starting the connector, please make sure that the configurations in
 ``etc/kafka-connect-hdfs/quickstart-hdfs.properties`` are properly set to your configurations of
-Hadoop, e.g. ``hdfs.url`` points to the proper HDFS and using FQDN in the host. Then run the
-following command to start Kafka connect with the HDFS connector::
+Hadoop, e.g. ``hdfs.url`` points to the proper HDFS and using FQDN in the host. Then start connector by loading its
+configuration with the following command:
+
+.. sourcecode:: bash
+
+   $ confluent load hdfs-sink -d etc/kafka-connect-hdfs/quickstart-hdfs.properties
+   {
```

Comment:
I had the same thought. Problem is that we list the output in almost every command. Feels a bit sub-optimal to separate command and output with a sentence everywhere. 

The question is, won't the reader understand what's happening after a couple commands? Pasting the whole thing will generate command line errors for sure. Let me know.